### ISSUE TITLE
Chrome issue

### ISSUE THREAD

**Issue by SirbitoX** [ichord/At.js#299]
Hi,

I'm using at.js on windows 7/chrome : Version 43.0.2357.124 m
When I try to scroll the loaded list, it disappears. But it works fine in Firefox.
Any one knows the reason??

**Comment by ichord** [comment]
It must be because of the input scrolling. I can't reappear it.
Please try on Mac/Chrome if you could, thanks.

**Comment by SirbitoX** [comment]
In fact when I click on sidebar in chrome, the list disappears.

**Comment by ichord** [comment]
Click on anywhere outside of the list will make the list disappears; It's not an issue.

**Comment by SirbitoX** [comment]
@ichord did you read my comment at all??
I said when you try to scroll the loaded list by clicking on SIDEBAR of THE LIST it disappears not outside of the list. SCROLL-BAR is a part of the loaded list and INSIDE it.

**Comment by saeedsq** [comment]
:-1: 
@SirbitoX is right.
It is a bug, You should fix it.

**Comment by ichord** [comment]
@SirbitoX 
First time, you said you try to scroll the loaded list, but you didn't tell how to.
And then you said 
> In fact when I click on sidebar in chrome
whose sidebar?? Browser ? textarea? list view?

This one is way better.
>I said when you try to scroll the loaded list by clicking on SIDEBAR of THE LIST it disappears not outside of the list. SCROLL-BAR is a part of the loaded list and INSIDE it.

I will be extremely grateful if you could show me an gif showing what happened.

Please tell me more if i misunderstand anything, this  :-1: is not going to help. @saeedsq

**Comment by SirbitoX** [comment]
@ichord You are right. My first comment is not precise enough. But when I'm talking about the loaded list, definitely I mean the sidebar of it.
I don't have any tool to create a gif file. I try to describe it precisely again if it helps:
* First of all add some css to strict the height of the loaded list:
```css
.ui-autocomplete {
    max-height: 200px;
    overflow-y: scroll;
}
```
* Then use "Chrome" browser. 
When the list appears try to scroll the list to choose an item in the bottom of the list and you would see that the loaded list disappears. This issue doesn't occur in the other browsers.

**Comment by Edward-Shaw** [comment]
interesting talk yum:

**Comment by tumetus** [comment]
I am experiencing this bug with IE11. Any updates on fixing this? Here is a gif about it.

![atjs_dropdown_bug](https://cloud.githubusercontent.com/assets/3472062/18428913/505970f6-78d9-11e6-91fe-1480ed78d09d.gif)

**Comment by sampatbadhe** [comment]
I am also facing same issue on Safari and Mozila as well.

**Comment by aluchenitser** [comment]
Also experiencing on IE11. Identical problem to tumetus' animated GIF.

**Comment by agusain** [comment]
@ichord this library is incredible.

commenting line `return c.view.hide(e, c.getOpt("displayTimeout")) ` in 'blur.atwhoInner' event for js file did the trick for me.

I don't know why clicking on scroll, the control  goes to 'blur.atwhoInner' event.

**Comment by isikfsc** [comment]
thanks @agusain, it has solved my problem.

**Comment by gowritumma** [comment]
Changed the bootstrap typeahead version and it worked.

bootstrap3-typeahead.min.js
